npm_and_yarn/spec/fixtures/projects/npm6/ghpr_no_hash_pinning/package.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "hypixel-translators",
```

Comment:
This would be good to strip down to basic example, maybe just including `discord.js` dependency. An easy way of generating an example with npm is running `yes "" | npm init && npm install github:discordjs/discord.js#ab82cafcde0ee259a32ef14303c1b4a64dea8fae --save --lockfile-only`, if you then remove the `#sha` from `package.json` and re-run `npm install` it should then update the lockfile `from` to match keeping the old sha `version`

Reply:
You shouldn't need to install with the #sha to begin with though?

Reply:
> You shouldn't need to install with the #sha to begin with though?

I think we want to make sure the installed sha is old and can be updated

Reply:
I updated the fixture to make sure that the declared constraint isn't pinned to a sha and that the `from` field in the lockfile matches this constraint. Please let me know if I've missed anything else.